doc/help/elektra-algorithm.md

```diff
@@ -138,7 +138,7 @@ to determine if they are responsible for a key or not.	Consequently, it
 can happen that more than one backend delivers a key with the same name.
 
 `kdbGet()` ensures that a key is uniquely identified by its name.
-Elektra's core will pop keys that are outside of the backend's
+Elektra's core will drop keys that are outside of the backend's
```

Comment:
Should be fixed now.

BTW: This is the first time I heard that there is a glossary. It might be a good idea to put the glossary somewhere after the table of contents. As it stands today it is at page 273 of the reference manual. That is not exactly a place where I would search for a glossary.